src/tidecli/tide_config.py

```diff
@@ -2,7 +2,11 @@
 
 # Configuration for the TIM /oauth
 CLIENT_ID = "oauth2_tide"
-TIM_URL = "http://localhost" if os.getenv("DEV") else "https://tim.jyu.fi"
+# Use DEV for localhost development address
+# For the tim url, use default https://tim.jyu.fi if the user does not supply custom url
+TIM_URL = (
+    "http://localhost" if os.getenv("DEV") else os.getenv("URL", "https://tim.jyu.fi")
```

Comment:
`"URL"` feels too vague (which URL?).

```suggestion
    "http://localhost" if os.getenv("DEV") else os.getenv("TIM_URL", "https://tim.jyu.fi")
```